### PR TITLE
[MINOR][DOCS][R] Use actual version in SparkR Arrow guide for copy-and-paste

### PR DESCRIPTION
**Pull request by HyukjinKwon**
## What changes were proposed in this pull request?

To address https://github.com/apache/spark/pull/24506#discussion_r280964509

## How was this patch tested?

N/A

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
cc @felixcheung

File: docs/sparkr.md [hunk 1]
```diff
@@ -666,10 +666,10 @@ Apache Arrow is an in-memory columnar data format that is used in Spark to effic
 Currently, Arrow R library is not on CRAN yet [ARROW-3204](https://issues.apache.org/jira/browse/ARROW-3204). Therefore, it should be installed directly from Github. You can use `remotes::install_github` as below.
 
 ```bash
-Rscript -e 'remotes::install_github("apache/arrow@TAG", subdir = "r")'
+Rscript -e 'remotes::install_github("apache/arrow@apache-arrow-0.12.1", subdir = "r")'
 ```
 
-`TAG` is a version tag that can be checked in [Arrow at Github](https://github.com/apache/arrow/releases). You must ensure that Arrow R package is installed and available on all cluster nodes. The current supported version is 0.12.1.
+`apache-arrow-0.12.1` is a version tag that can be checked in [Arrow at Github](https://github.com/apache/arrow/releases). You must ensure that Arrow R package is installed and available on all cluster nodes. The current supported version is 0.12.1.
```
Comment (felixcheung) [inline docs/sparkr.md]: btw, is there any problem if I have a newer version of arrow, like 0.13?

**Review by felixcheung — APPROVED** [review]

**Review by dongjoon-hyun — APPROVED** [review]

File: docs/sparkr.md (hunk 1 above)
Comment (dongjoon-hyun) [reply docs/sparkr.md]: I have the same curiosity. :)
Since this is orthogonal and may need `pom.xml`, I'll merge this first.

**Comment by dongjoon-hyun** [comment]
Merged to master.

**Pull request closed without merging**